storage/remote/queue_manager.go

```diff
@@ -33,6 +33,33 @@ const (
 	dropped = "dropped"
 )
 
+var (
+	sentSamplesTotalDesc = prometheus.NewDesc(
```

Comment:
These descs are now only used in the `Describe()` method, but not anywhere else?